webpack/vellumUtils.js

```diff
@@ -63,12 +63,20 @@ const getDebugRule = function () {
     if (!config) {
         return {};
     }
+    const debugDir = getDebugDir(),
+        hqDir = path.resolve(__dirname, '..');
+    const rules = config.module.rules.map(function (rule) {
+        if (rule.loader) {
+            rule.loader = rule.loader.replace(hqDir, debugDir);
```

Comment:
What do you think of transforming `hqDir` into a regex so we can do a stricter (_startswith_) match? Something like
```suggestion
    const debugDir = getDebugDir();
    const hqDir = path.resolve(__dirname, '..');
    const hqDirPrefix = new RegExp('^' + RegExp.escape(hqDir));
    const rules = config.module.rules.map(function (rule) {
        if (rule.loader) {
            rule.loader = rule.loader.replace(hqDirPrefix, debugDir);
```